### PR TITLE
Add typeahead.js-bootstrap-css@1.2.1 w/ git auto-update

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #10254
Related issue(s): # #

@cdnjs/intern3 Please help me review this PR. Thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/bassjobsen/typeahead.js-bootstrap-css
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/typeahead.js-bootstrap-css
 * License and its reference: MIT, [ref](https://github.com/bassjobsen/typeahead.js-bootstrap-css/blob/master/LICENSE.md)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 16
   - Count of stars: 164
   - Count of forks: 103
 * NPM download stats (optional):
   - Downloads in the last day: 140
   - Downloads in the last week: 633
   - Downloads in the last month: 2,695

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by hare1039** [comment]
@pvnr0082t Please help me reviewing this PR. Thanks.

File: ajax/libs/typeahead.js-bootstrap-css/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bassjobsen/typeahead.js-bootstrap-css.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "typeaheadjs.css"
```
Comment (pvnr0082t) [inline ajax/libs/typeahead.js-bootstrap-css/package.json]: @hare1039 You should make it more flexible. Try to exclude `.less`

**Comment by hare1039** [comment]
@pvnr0082t Sorry for the careless mistake I made. I fix the commit message. Please review this PR again. Thanks.

File: ajax/libs/typeahead.js-bootstrap-css/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bassjobsen/typeahead.js-bootstrap-css.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "typeaheadjs.!(*.less)"
```
Comment (pvnr0082t) [inline ajax/libs/typeahead.js-bootstrap-css/package.json]: @hare1039 Have you tried this config? It will fetch `*.less` file.

**Comment by hare1039** [comment]
@pvnr0082t I fix the commit message and package.json. Please review this PR again. Thanks.

**Comment by pvnr0082t** [comment]
@hare1039 Please rebase it onto the latest master branch. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t I've rebased to master branch.

**Pull request merged**